src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -237,10 +237,16 @@ static void upgradeImportFileAndDirePatterns(JabRefPreferences prefs, Preference
             String[] oldStylePatterns = new String[] {"\\bibtexkey",
                                                       "\\bibtexkey\\begin{title} - \\format[RemoveBrackets]{\\title}\\end{title}"};
             String[] newStylePatterns = new String[] {"[bibtexkey]",
-                                                      "[bibtexkey] - [fulltitle]"};
+                                                      "[bibtexkey] - [title]"};
+
+            String[] oldDisplayStylePattern = new String[] {"bibtexkey", "bibtexkey - title",};
+
             for (int i = 0; i < oldStylePatterns.length; i++) {
                 migrateFileImportPattern(oldStylePatterns[i], newStylePatterns[i], prefs, mainPrefsNode);
             }
+            for (int i = 0; i < oldDisplayStylePattern.length; i++) {
```

Comment:
Do we really need a migration here? As this bug was introduced in 5.0 I doubt that many users are affected...

Reply:
It looks weird as it will display a wrong value in the field next to the combobox (then selectable)